tensorflow/workspace2.bzl

```
@@ -207,6 +207,8 @@ def _tf_repositories():
             "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch",
             "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch",
             "//third_party/mkl_dnn:onednn_acl_threadpool_scheduler.patch",
+            "//third_party/mkl_dnn:onednn_reorder_padded.patch",
```

To be consistent to #61295,
```suggestion
            "//third_party/mkl_dnn:onednn_acl_reorder_padded.patch",
```